client/src/components/SearchVideo.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+
+
+
+
+const SearchVideo = (prop) => {
+    let videos = prop.videos;
+  const [videoSearch, setvideo] = useState("");
+  
+  
+  const [newvideo, setNewVideo] = useState([{}]);
+    const handleChange = (e) => {
+        setvideo(e.target.value.toLowerCase());
+
+        setNewVideo(videos.filter(video => video.title.toLowerCase().includes(videoSearch) || 
+        video.url.toLowerCase().includes(videoSearch)))
+      }
```

Comment:
your search works really well! At the moment you are filtering the videos every time handleChange is called, but you are not showing the filtered results until the search button is clicked. You could strip your filter function out of here & only filter the results once the 'Search Video' button has actually been clicked

Reply:
done